Sources/App/Views/Keyword/KeywordShow+View.swift

```diff
@@ -66,8 +79,15 @@ enum KeywordShow {
                 )
             )
         }
-    }
 
+        private func packageCollectionLimitNote() -> Node<HTML.BodyContext> {
+            guard model.totalPackageCount > Constants.maxKeywordPackageCollectionCount else { return .empty }
+            return .p(
+                .strong("Note: Package collections are limited to a maximum of \(Constants.maxKeywordPackageCollectionCount) packages. If a keyword has more than \(Constants.maxKeywordPackageCollectionCount) packages, only the top \(Constants.maxKeywordPackageCollectionCount) packages will be included based on their package score."
```

Comment:
Apologies, I meant for the strong to be only around the "Note:"

```suggestion
                .strong("Note: "),
                .text("Package collections are limited to a maximum of \(Constants.maxKeywordPackageCollectionCount) packages. If a keyword has more than \(Constants.maxKeywordPackageCollectionCount) packages, only the top \(Constants.maxKeywordPackageCollectionCount) packages will be included based on their package score."
```

Reply:
Sure thing, updated!